ipn/ipnlocal/local.go

```diff
@@ -6583,9 +6593,12 @@ var ErrCannotSuggestExitNode = errors.New("unable to suggest an exit node, try a
 // Peers are selected based on having a DERP home that is the lowest latency to this device. For peers
 // without a DERP home, we look for geographic proximity to this device's DERP home.
 // b.mu.lock() must be held.
-func (b *LocalBackend) suggestExitNodeLocked() (response apitype.ExitNodeSuggestionResponse, err error) {
+func (b *LocalBackend) suggestExitNodeLocked(netMap *netmap.NetworkMap) (response apitype.ExitNodeSuggestionResponse, err error) {
+	// netMap is an optional netmap to use that overrides b.netMap (needed for SetControlClientStatus before b.netMap is updated). If netMap is nil, then b.netMap is used.
```

Comment:
Oh this should be in the docstring above the `func` line. It's also kind of long, so wrap it like the rest of the docstring